static/src/javascripts/projects/common/modules/commercial/creatives/fluid250.js

```diff
@@ -39,11 +59,21 @@ define([
                     '<iframe width="409px" height="230px" src="' + this.params.videoURL + '?rel=0&amp;controls=0&amp;showinfo=0&amp;title=0&amp;byline=0&amp;portrait=0" frameborder="0" class="fluid250_video fluid250_video--desktop fluid250_video--vert-pos-' + this.params.videoPositionV + ' fluid250_video--horiz-pos-' + this.params.videoPositionH + '" style="' + leftPosition + rightPosition + '"></iframe>' : ''
             };
 
-        $.create(template(fluid250Tpl, _.merge(this.params, templateOptions, videoDesktop))).appendTo(this.$adSlot);
+        this.$fluid250 = $.create(template(fluid250Tpl, _.merge(this.params, templateOptions, videoDesktop))).appendTo(this.$adSlot);
 
         if (this.params.trackingPixel) {
             this.$adSlot.before('<img src="' + this.params.trackingPixel + this.params.cacheBuster + '" class="creative__tracking-pixel" height="1px" width="1px"/>');
         }
+
+        if (Fluid250.hasScrollEnabled && (this.params.backgroundImageScroll !== 'normal' || this.params.layerOneImageScroll !== 'normal'  || this.params.layerTwoImageScroll !== 'normal'  || this.params.layerThreeImageScroll !== 'normal')) {
```

Comment:
Too long line, I would suggest writing something like:
 if (Fluid250.hasScrollEnabled && (this.params.backgroundImageScroll !== 'normal' ||
    this.params.layerOneImageScroll !== 'normal' ||
    this.params.layerTwoImageScroll !== 'normal' ||
  (...)